2018/sprints.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+order: 1
+title: "2018"
+subtitle: "Sprints"
+titleDisplay: "Sprints"
+excerpt:
+venueId:  
+date:
+startTime:
+endTime:
+---
+
+Monday–Wednesday • 16–18
+
+Modelled after [PyCon](https://us.pycon.org/2018/community/sprints/), we are providing space for project sprints following the conference. Sprints provide an opportunity for people interested in a project to come together to learn and create in a truely Do It With Others (DIWO) environment. The goal is to make friends and build lasting relationships through working together to share knowledge, build features, hack on prototypes, and polish resources.
+
+We'll take care of providing space from 10am - 9:30pm Monday through Wednesday with hi-speed internet, power bars, post-its, and markers.
+
+## Join a Sprint
+
+Add your name to the list and make sure to watch the sprint pitches on Sunday at the conference, the project lead will be there providing an overview.
+
+| Project Name | Project Lead | Description and Goals | Attending |
+|:-------------|:-------------|:----------------------|:----------|
+|              |              |                       |           |
+
+
+## Propose a Sprint
+
+Anyone can propose a Sprint in the lead up to and at the conference. If a new idea comes up during Our Networks and you feel a Sprint may be the right format to test / kickstart the project, we encourage people pitch a Sprint.
+
+Pitching a Sprint requires a short description about the idea along with the following information:
```

Comment:
need to make clear you PR this repo (and this page to add yr deets) Is there a PR template for that? (no right?)